src/index.js

```diff
@@ -11,25 +7,27 @@ const moment = require('moment');
 
 let selectionBox = document.querySelector('.selection-box');
 let calculateCost = document.querySelector('.calculate-cost');
+let bookTripButton = document.querySelector('.book-requested-trip');
 
-let tripsToDisplay, allDestinations, user;
+let requestedTripObject, allDestinations, user;
```

Comment:
```suggestion
let requestedTrip, allDestinations, user;
```

Reply:
My b. I was struggling to get that fetch to work at first. Definitely making an issue for naming techniques.